server/item/enchantment/vanishing.go

```diff
@@ -0,0 +1,52 @@
+package enchantment
+
+import (
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// Vanishing is an enchantment that causes the item to disappear on death.
+type Vanishing struct{}
+
+// Name ...
+func (Vanishing) Name() string {
+	return "Vanishing"
+}
+
+// MaxLevel ...
+func (Vanishing) MaxLevel() int {
+	return 1
+}
+
+// Cost ...
+func (Vanishing) Cost(int) (int, int) {
+	return 15, 65
```

Comment:
These costs are incorrect - `25` for minimum and `50` for maximum according to Java Edition.